### PR TITLE
prepare new release 3.2.1

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Related to #3654 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- No test necessary here

### CODE REVIEW

**Comment by PhilippeAccorsi** [comment]
Wait discuss with @lebouquetin to choose space type order

File: changelog.md [hunk 1]
```diff
@@ -1,11 +1,35 @@
 
+
+## 3.2.1 / 2020-11-04
+
+### New Features
+
+- Full implementation of spaces access types: open, on request and confidential (spaces available in Tracim < 3.2.1 are considering like type confidential)
+- Allow user to discover and join spaces
```
Comment (raphj) [inline changelog.md:8]:
```suggestion
- Allow users to discover, join and leave spaces
```

File: changelog.md [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+- New plugins available:
+  - Adds every new user to all open spaces and to every newly created open space. You need to activate this plugin `tracim_backend_autoinvite` (documentation avialable [here](backend/official_plugins/README.md))
+  - Recursively adds new members of space to its parents with the default user role of each space. You need to activate this plugin `tracim_backend_parent_access` (documentation avialable [here](backend/official_plugins/README.md))
```
Comment (raphj) [inline changelog.md:11]:
```suggestion
  - Recursively adds new members of a space to its parents with the default user role of each space. You need to activate this plugin `tracim_backend_parent_access` (documentation avialable [here](backend/official_plugins/README.md))
```

File: changelog.md [hunk 3]
```diff
@@ -1,11 +1,35 @@
 
+
+## 3.2.1 / 2020-11-04
+
+### New Features
+
+- Full implementation of spaces access types: open, on request and confidential (spaces available in Tracim < 3.2.1 are considering like type confidential)
```
Comment (grignards) [inline changelog.md:7]:
```suggestion
- Full implementation of spaces access types: open, on request and confidential. Spaces created with Tracim < 3.2.1 are all of type confidential and retain their current access behavior.
```

File: changelog.md [hunk 4]
```diff
@@ -1,11 +1,35 @@
...
+- Allow user to discover and join spaces
+- New plugins available:
+  - Adds every new user to all open spaces and to every newly created open space. You need to activate this plugin `tracim_backend_autoinvite` (documentation avialable [here](backend/official_plugins/README.md))
```
Comment (grignards) [inline changelog.md:10]:
```suggestion
  - Adds every new user to all open spaces and to every newly created open space. You need to activate this plugin `tracim_backend_autoinvite` (documentation available [here](backend/official_plugins/README.md))
```

File: changelog.md [hunk 5]
```diff
@@ -1,11 +1,35 @@
...
+  - Recursively adds new members of space to its parents with the default user role of each space. You need to activate this plugin `tracim_backend_parent_access` (documentation avialable [here](backend/official_plugins/README.md))
+  - When a user is removed from a space, this plugin recursively removes this user from the children of this space. You need to activate this plugin `tracim_backend_child_removal` (documentation avialable [here](backend/official_plugins/README.md))
+- Group vendor dependencies to make faster and smaller frontend builds (feature for developers and production)
```
Comment (raphj) [inline changelog.md:13]: It also makes Tracim way more lightweight, enabling usage of Tracim on cheap / older device such as my cheap old phone and reducing bandwidth consumption when loading tracim (making Tracim faster to load on small connections)

File: changelog.md (hunk 2 above)
Comment (grignards) [inline changelog.md:11]:
```suggestion
  - Recursively adds new members of space to its parents with the default user role of each space. You need to activate this plugin `tracim_backend_parent_access` (documentation available [here](backend/official_plugins/README.md))
```

File: changelog.md [hunk 6]
```diff
@@ -1,11 +1,35 @@
...
+## 3.2.1 / 2020-11-04
+
+### New Features
+
+- Full implementation of spaces access types: open, on request and confidential (spaces available in Tracim < 3.2.1 are considering like type confidential)
+- Allow user to discover and join spaces
+- New plugins available:
+  - Adds every new user to all open spaces and to every newly created open space. You need to activate this plugin `tracim_backend_autoinvite` (documentation avialable [here](backend/official_plugins/README.md))
+  - Recursively adds new members of space to its parents with the default user role of each space. You need to activate this plugin `tracim_backend_parent_access` (documentation avialable [here](backend/official_plugins/README.md))
+  - When a user is removed from a space, this plugin recursively removes this user from the children of this space. You need to activate this plugin `tracim_backend_child_removal` (documentation avialable [here](backend/official_plugins/README.md))
```
Comment (grignards) [inline changelog.md:12]:
```suggestion
  - When a user is removed from a space, this plugin recursively removes this user from the children of this space. You need to activate this plugin `tracim_backend_child_removal` (documentation available [here](backend/official_plugins/README.md))
```

File: changelog.md (hunk 3 above)
Comment (raphj) [inline changelog.md:7]:
```suggestion
- Full implementation of spaces access types: open, on request and confidential (the type of spaces created on Tracim < 3.2.1 is confidential)
```

File: changelog.md (hunk 5 above)
Comment (raphj) [reply changelog.md:13]:
```suggestion
- vendor dependencies are grouped, making Tracim faster to load on small connections and older devices, and more lightweight on memory consumption. For developers and sys admins, this also makes Tracim build faster.
```

File: changelog.md [hunk 7]
```diff
@@ -23,8 +47,8 @@
 - Renaming `Text document` app to `Note` (issue [#3621](https://github.com/tracim/tracim/issues/3621))
 - Renaming `Shared space` to `Space` (issue [#3582](https://github.com/tracim/tracim/issues/3582))
 - Webdav (issue [#3625](https://github.com/tracim/tracim/issues/3625)):
-  - It is no longer possible to show two spaces with the same name in webdav. If this case exist on Tracim, webdav show the first space id.
-  - It is no longer possible to create/modify a space from webdav access
+  - It is no longer possible to show two spaces with the same name in WebDAV. If two spaces at the same place have the same, will only show the first oldest space.
```
Comment (raphj) [inline changelog.md:50]:
```suggestion
  - It is no longer possible to show two spaces with the same name in WebDAV. If two spaces at the same place have the same name, only the oldest space will be shown.
```

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md:11]: Can you check if this fix need to be added also in officla plugin README ? (if yes, I can change readme in this MR)

File: changelog.md (hunk 3 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Modification done.

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md]: This is already done in plugin readme. I had this fix in chnagelog.

**Review by gbonaspetti — APPROVED** [review]

**Review by grignards — APPROVED** [review]

**Pull request merged**